docs/COPILOT.md

```diff
@@ -0,0 +1,78 @@
+
+# GitHub Copilot 
+
+> [!NOTE]
+> This document is based on YUMEMI Inc.'s internal document "[Policies, Rules, and Guidelines for the Use of ChatGPT and Other AI Services](https://www.notion.so/yumemi/ChatGPT-AI-f951c2e2542e4db5af72386329e23155)".
+> Please check that guideline at the beginning if you are within YUMEMI.
+
+## Risks of Generated AI in general
+Generated AI in general has the following risks. You need to be fully aware of them with related parties when you use them.
+
+### Risks under copyright law
+Automatically generated source code may infringe on the copyright of others' source code.
+
+### Security Risks
+Code may be generated with insecure coding patterns or bugs, use of outdated APIs or libraries, or other security risks.
+
+### Privacy Risks
+There is concern that sensitive or personal information could be learned and displayed to third parties.
+You should be aware of what information those tools collect from your project and use for learning.
+
+## Risks of GitHub Copilot
+GitHub Copilot can be used by setting up a subscription.
+There are two types of subscriptions: personal and organizational (Organization or Enterprise) as shown below. Note that each has a different privacy policy.
+- [GitHub Copilot Individual](https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot-individual)
+- [GitHub Copilot Business](https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot-business)
+
+### Risk as an IDE extension
+Note that the extent to which GitHub Copilot extensions are enabled depends on the IDE.
+VSCode and Xcode allow you to enable it on a per-project basis, but other IDEs do not allow you to control it on a per-project basis, so you must be careful not to accidentally run Copilot.
+
+## How to use Copilot
+### VSCode
+1. install [GitHub Copilot extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) to VSCode
+2. disable the Copilot extension so that Copilot will not work throughout VSCode
+3. open any workspace and enable the Copilot extension in that workspace only
+4. Add the following GitHub Copilot configuration to the workspace settings (`.vscode/settings.json`)
+  
+```jsonc
+{
+  "github.copilot.enable": {
+    // Specify which languages you want to enable Copilot for.
+    "*": false,
+    "dart": true
+  },
+  "github.copilot.editor.enableCodeActions": true, 
+  "github.copilot.editor.enableAutoCompletions": true
+}
+```` 
+
+5. (Optional) Disable Copilot in the overall VSCode settings (`settings.json`)
+
+```json
+{
+  "github.copilot.enable": {
+    "*" : false
+  },
+  "github.copilot.editor.enableCodeActions": false,
+  "github.copilot.editor.enableAutoCompletions": false
+}
+```
+
+VSCode - Programming Languages Overview
+https://code.visualstudio.com/docs/languages/overview
+
+List of languages supported by VSCode
+https://code.visualstudio.com/docs/languages/identifiers
+
```

Comment:
![imo-badge](https://img.shields.io/badge/review-imo-skyblue.svg) 
Unlike the template project, the settings will not be reset, so I thought specific setup instructions might not be necessary.

![ask-badge](https://img.shields.io/badge/review-ask-yellowgreen.svg) 
How about just providing a reference link?

Reply:
Fixed 0d2a21c4e8d1083ed9c2d905086b67cd14f3592d